.eslintrc.cjs

```diff
@@ -22,4 +22,16 @@ module.exports = {
     'object-shorthand': 'error',
     'prefer-arrow-callback': 'error',
   },
+  overrides: [
+    {
+      files: ['perf/*.js'],
+      env: {
+        browser: true,
+      },
+      rules: {
+        '@typescript-eslint/explicit-module-boundary-types': 'off',
```

Comment:
I liked this rule. How was it getting in your way?

FWIW, You can always disable a rule on a specific line with a comment.

Reply:
It just fires for all of the unannotated JS code in `perf/*.js`, now that that falls under the rules in `.eslintrc.cjs`; this exclusion is only active for those files.